src/main/java/org/fcrepo/spec/testsuite/versioning/LdprvHttpPut.java

```diff
@@ -50,35 +60,57 @@ public void ldprvMustSupportPUT(final String uri) {
                                         ps);
 
         //create an LDPRv using a put
+        final Response response = putVersionedResourceUnverified(uri, info);
+        assertEquals(response.getStatusCode(), 201);
         //verify that it is a TimeGate and has a TimeMap
-
+        final Response getResponse = doGet(getLocation(response));
+        confirmPresenceOfLinkValue(ORIGINAL_RESOURCE_LINK_HEADER, getResponse);
+        confirmPresenceOfLinkValue(TIME_GATE_LINK_HEADER, getResponse);
     }
 
     /**
      * 4.1.2-B
      *
      * @param uri The repository root URI
      */
-    //@Test(groups = {"MUST"})
+    @Test(groups = {"MUST"})
     @Parameters({"param1"})
     public void ldprvMustSupportPUTForExistingResources(final String uri) {
         final TestInfo info = setupTest("4.1.2-B", "ldprvMustSupportPUTForExistingResources",
                                         "Must support PUT for updating existing LDPRvs",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldprv-put",
                                         ps);
         //create an LDPRv
+        final Response response = putVersionedResourceUnverified(uri, info);
+        assertEquals(response.getStatusCode(), 201);
+
+        final String resourceUri = getLocation(response);
+        final Response getResponse = doGet(resourceUri);
```

Comment:
We probably want to specify an `Accept` header of `text/turtle`, since some implementations may return a different RDF serialization by default.